doc/cffi-manual.texinfo

```diff
@@ -5559,12 +5557,118 @@ CFFI> (foreign-funcall-pointer (foreign-symbol-pointer "abs") ()
 @seealso{defcfun} @*
 @seealso{foreign-funcall}
 
+@c ===================================================================
+@c FOREIGN-FUNCALL-VARARGS
+
+@page
+@node foreign-funcall-varargs, foreign-funcall-pointer-varargs, foreign-funcall-pointer, Functions
+@heading foreign-funcall-varargs
+@subheading Syntax
+@Macro{foreign-funcall-varargs name-and-options (fixed-arguments) &rest arguments @res{} return-value}
+
+fixed-arguments ::= @{ arg-type arg @}* [return-type]
+arguments ::= @{ arg-type arg @}* [return-type]
+name-and-options ::= name | ( name &key library convention)
```

Comment:
The `foreign-funcall` node had some layout issues. I've fixed those in f1b3a5c10aeedfdd9f5913d0bd6d45db5f22f269 and fixed up your commit with same fixes in 9e2176adff97c9564228c084bbe48ea94c8a9725.